mava/systems/tf/madqn/system.py

```diff
@@ -255,7 +258,7 @@ def __init__(  # noqa
                 agents,
                 self._network_sampling_setup,  # type: ignore
             )
-
+        print(self._agent_net_keys)
```

Comment:
I assume this is just for debug?

Needs to be removed before we merge.

Reply:
ah yes I have removed it